src/components/organisms/sections/ContactForm/index.jsx

```diff
@@ -21,6 +20,12 @@ import {
 //> Images
 // To be added
 
+//> Redux
+// Connect
+import { connect } from "react-redux";
+// Actions
```

Comment:
Missing new line between internal and external imports.

Reply:
These are both Redux based imports under the //> Redux header.